### PR TITLE
Check for no wrap fix

### LINKED ISSUE #9521: Bidirectional FocusZone with checkForNoWrap not working correctly.
There's a bug in FocusZone where if _active element is the left most element in any row (except top row)_ **AND** _the top row is not in scroll view_, pressing left arrow button moves focus to first element in the top row rather than doing nothing.

This works fine when all the elements are visible on the screen.

You can see this in action here. Click on 1st item in the 4th row and press left arrow key. Instead of doing nothing, the first 1st item in 1st row gets selected.

https://codepen.io/cvirendra/pen/dBOjZO

### PR DESCRIPTION
**Pull request by cvirendra**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9521

#### Description of changes

Convert strings to floating point before using.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9542)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx [hunk 1]
```diff
@@ -224,7 +224,7 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
           // root props has been deprecated and should get removed.
           // it needs to be marked as "any" since root props expects a div element, but really Tag can
           // be any native element so typescript rightly flags this as a problem.
-          ...rootProps as any
+          ...(rootProps as any)
```
Comment (cvirendra) [inline packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx]: This change seems to be made automatically. I'm not sure what's causing this.

File: common/changes/office-ui-fabric-react/checkForNoWrapFix_2019-06-21-05-14.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixed keyboard navigation when using checkForNoWrap",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/checkForNoWrapFix_2019-06-21-05-14.json]:
```suggestion
      "comment": "FocusZone: fixed keyboard navigation when checkForNoWrap prop enabled.",
```

File: packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx (hunk 1 above)
Comment (KevinTCoughlin) [reply packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx]: Its caused automatically by the Prettier configuration checked into the repo.

**Comment by aftab-hassan** [comment]
@JasonGore could you please take a look at this community contribution and help review. 

@cvirendra thank you for opening your first pull request on our repo!

**Pull request merged**